File: 4177-ReverseStringPrefix/4177-ReverseStringPrefix.cpp
// Last updated: 3/30/2026, 2:10:48 PM
class Solution {
public:
    string reversePrefix(string s, int k) {
        string substring = s.substr(0,k);
        reverse(substring.begin(),substring.end());
        return substring + s.substr(k,s.size());
    }
};

File: 4177-ReverseStringPrefix/4177-ReverseStringPrefix.java
// Last updated: 3/30/2026, 2:10:44 PM
class Solution {
    public String reversePrefix(String s, int k) {
        String substring = s.substring(0,k);
        StringBuilder sb = new StringBuilder(substring);
        return sb.reverse().toString() + s.substring(k,s.length());
    }
}